### DISCUSSION TITLE
issues is pending so i post it here

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#615] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
killcurly is not working on version 112 and up. specificly chrome://settings/signOut.
the banner that is supposed to appear does not appear so i cant "sign out".

**Comment by funkymokey** [comment]
my computer is on 108 and i have never seen killcurly actually work

**Reply by Kazuma-Itsuchi** [reply]
It used to work for me. It stopped on the new update.

**Reply by Aka-but-nice** [reply, marked as answer]
Yep, they patched it, but it's no longer working, they've finally removed the forgotten Chrome URL.

**Comment by synapt0** [comment]
yeah same problem here

**Comment by plmnbvcx** [comment]
works for me?

**Reply by MilkMaker23** [reply]
everything works but it does not do anything
